backend/validators/llm.py

```
@@ -0,0 +1,141 @@
+import signal
+import os
+import sys
+import dataclasses
+
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+from .base import *
```

Ruff (0.11.9)

`from .base import *` used; unable to detect undefined names

(F403)